src/java/org/apache/cassandra/db/repair/CassandraTableRepairManager.java

```diff
@@ -87,7 +87,9 @@ public synchronized void snapshot(String name, Collection<Range<Token>> ranges,
                                                                     !sstable.metadata().isIndex() && // exclude SSTables from 2i
                                                                     new Bounds<>(sstable.getFirst().getToken(), sstable.getLast().getToken()).intersects(ranges);
 
-                    SnapshotOptions options = SnapshotOptions.systemSnapshot(name, SnapshotType.REPAIR, predicate, cfs.getKeyspaceTableName()).ephemeral().build();
+                    SnapshotOptions options = SnapshotOptions.systemSnapshot(name, SnapshotType.REPAIR, predicate, cfs.getKeyspaceTableName())
```

Comment:
Do we not have to inject `force` into `SnapshotOptions options` somewhere?

Reply:
We now use `ephemeral` as the sole criterion in this change, making the `force` supplied in `snapshot` orphan. If that is the intention, then we should clean up the `force` supplied in `snapshot` API.